src/test/java/org/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -254,1144 +64,106 @@ public void setUp() throws IOException, COSVisitorException {
-                        + "  year = {2005}," + "\n" + "  owner = {oezbek}," + "\n" + "  timestamp = {2006.05.29},"
-                        + "\n" + "  url = {http://james.howison.name/publications.html}" + "\n" + "}"));
-
-        Collection<BibEntry> c = original.getDatabase().getEntries();
-        Assert.assertEquals(1, c.size());
-
-        BibEntry e = c.iterator().next();
-
-        XMPUtil.writeXMP(pdfFile, e, original.getDatabase(), xmpPreferences);
-
-        List<BibEntry> l = XMPUtil.readXMP(pdfFile.getAbsoluteFile(), xmpPreferences);
-        Assert.assertEquals(1, l.size());
-        BibEntry x = l.get(0);
-
-        Assert.assertEquals(AuthorList.parse("Crowston, K. and Annabi, H. and Howison, J. and Masango, C."),
-                AuthorList.parse(x.getField("author").get()));
-    }
-
-    @Test(expected = EncryptedPdfsNotSupportedException.class)
-    public void expectedEncryptionNotSupportedExceptionAtRead() throws IOException {
-        try (InputStream is = XMPUtilTest.class.getResourceAsStream("/pdfs/encrypted.pdf")) {
-            XMPUtil.readXMP(is, xmpPreferences);
+        // create a default PDF
+        File pdfFile = tempFolder.newFile("JabRef.pdf");
+        try (PDDocument pdf = new PDDocument()) {
+            // Need a single page to open in Acrobat
+            pdf.addPage(new PDPage());
+            pdf.save(pdfFile.getAbsolutePath());
         }
-    }
-
-    @Test(expected = EncryptedPdfsNotSupportedException.class)
-    public void expectedEncryptionNotSupportedExceptionAtWrite() throws IOException, TransformerException {
-        XMPUtil.writeXMP("src/test/resources/pdfs/encrypted.pdf", t1BibtexEntry(), null, xmpPreferences);
-    }
-
-    /**
-     * A better testcase for resolveStrings. Makes sure that also the document information and dublin core are written
-     * correctly.
-     * <p/>
-     * Data was contributed by Philip K.F. Hölzenspies (p.k.f.holzenspies [at] utwente.nl).
-     *
-     * @throws IOException
-     * @throws TransformerException
-     */
-    @Test
-    public void testResolveStrings2() throws IOException, TransformerException {
-
-        try (BufferedReader fr = Files.newBufferedReader(Paths.get("src/test/resources/org/jabref/util/twente.bib"),
-                StandardCharsets.UTF_8)) {
-            ParserResult result = new BibtexParser(importFormatPreferences, fileMonitor).parse(fr);
-
-            Assert.assertEquals("Arvind", result.getDatabase().resolveForStrings("#Arvind#"));
 
-            AuthorList originalAuthors = AuthorList.parse(
-                    "Patterson, David and Arvind and Asanov\\'\\i{}c, Krste and Chiou, Derek and Hoe, James and Kozyrakis, Christos and Lu, S{hih-Lien} and Oskin, Mark and Rabaey, Jan and Wawrzynek, John");
+        // read a bib entry from the tests before
+        List<BibEntry> entries = XMPUtil.readXMP(XMPUtil.class.getResource("/org/jabref/logic/xmp/PD_metadata.pdf").toURI().getPath(), xmpPreferences);
```

Comment:
I think it is easier if you just create the entry by hand (and then write it, read and compare).